client/state/data-layer/wpcom/me/blocks/sites/index.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { READER_SITE_BLOCKS_REQUEST, READER_SITE_BLOCKS_RECEIVE } from 'state/action-types';
```

Comment:
Nit: would you like to sort the imports in alphabetical order? In addition to keeping things tidy I've found that it helps when reading the code.